public/src/components/channelManagement/epicTests/epicTestsForm.tsx

```diff
@@ -82,16 +82,27 @@ export interface EpicTest extends Test {
 
 type Props = InnerComponentProps<EpicTest>;
 
+const EPICS_THAT_ONLY_ALLOW_A_SINGLE_VARIANT: EpicType[] = ['ARTICLE_HOLDBACK', 'APPLE_NEWS'];
+const EPICS_THAT_DONT_ALLOW_VARIANT_SPLIT: EpicType[] = [
+  ...EPICS_THAT_ONLY_ALLOW_A_SINGLE_VARIANT,
+  'AMP',
+];
+
+export const isOnlyAllowedOneVariant = (epicType: EpicType): boolean =>
+  EPICS_THAT_ONLY_ALLOW_A_SINGLE_VARIANT.includes(epicType);
+
+export const isNotAllowedVariantSplit = (epicType: EpicType): boolean =>
+  EPICS_THAT_DONT_ALLOW_VARIANT_SPLIT.includes(epicType);
+
```

Comment:
This was a temporary solution to help with the logic. I'd advocate for removing this in the long run in favour of an `EpicConfig` option per epic type

Reply:
I like your suggestion of the `EpicConfig` 👍

Reply:
Would it be much work? It feels like we're half way there with this code!

Reply:
I've started work on it now - If it ends up being to much code I can always split it out, if not I'll just push it up to here as well

Reply:
I've pushed up all that refactor! I'll update the PR description tomorrow though!